composer.json

```diff
@@ -30,6 +30,7 @@
     },
     "require-dev": {
         "psr/log": "^1 || ^2 || ^3",
+        "psr/simple-cache": "^1.0 || ^2.0 || ^3.0",
```

Comment:
do we directly interact with simple-cache, or why do we need this here?

Reply:
The CachedClientFunctionalTest requires this else it did throw an exception that psr/simple-cache is not installed.

Reply:
ah, and otherwise it uses the deprecated doctrine cache. makes sense.

in 2.x, this is even a full requirement: https://github.com/jackalope/jackalope-doctrine-dbal/blob/2.x/composer.json#L26 not dev only